pkg/deploy/util/util.go

```diff
@@ -72,6 +72,17 @@ func LabelForDeploymentConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, config.LatestVersion)
 }
 
+// ConfigSelector constructs a label selector for the provided DeploymentConfig
+func ConfigSelector(config *deployapi.DeploymentConfig) labels.Selector {
+	label := config.Template.ControllerTemplate.Selector
```

Comment:
I'm sorry, I confused labels/annotations in my previous statements. To positively identify all deployments for a config, you need to look at the annotations on the RC's. Any RC with an annotation DeploymentConfigAnnotation=config.Name is a deployment. Now that labels can be namespaced, it'd be nice if you could rely on a label, but that's not currently the case. And since you can't do a query on annotations, I think you're stuck with a list/annotation filter on the client side.